### ISSUE TITLE
triangular arbitrage

### ISSUE THREAD

**Issue by onthetunnel** [StBlackwood/cryptohack#1]
hello , can i know this project can do triangular arbitrage on binance? Thanks You !

**Comment by StBlackwood** [comment]
No. For the triangular arbitrage to happen there should be a third exchange platform involved. Here the project exploits only the arbitrage between Koinex and Binance. I am open for suggestion to involve a third platform.

**Comment by onthetunnel** [comment]
thanks for your reply , you has misunderstand the triangular arbitrage , is not three exchange arbitrage. .. is trade on one exchange .. exp : BTC - XXX - ETH - BTC ...arbitrage between 3 step on pair...
